lux/vislib/altair/Histogram.py

```diff
@@ -45,48 +44,52 @@ def initialize_chart(self):
 
         x_min = self.vis.min_max[msr_attr.attribute][0]
         x_max = self.vis.min_max[msr_attr.attribute][1]
```

Comment:
You can write, x_min, x_max = self.vis.min_max[msr_attr.attribute]

Reply:
Great tip, thanks!

Reply:
Great tip, thanks!